telegram/ext/_callbackcontext.py

```diff
@@ -321,12 +301,12 @@ def from_update(
             if chat:
                 self._chat_id_and_data = (
                     chat.id,
-                    application.chat_data[chat.id],  # pylint: disable=protected-access
```

Comment:
how come this is not needed anymore?

Reply:
I was surprised too, maybe something in pylint internals stopped it from detecting this before? idk

Reply:
okay. interesting :D